components/tools/OmeroPy/src/omero/util/sessions.py

```diff
@@ -127,7 +127,9 @@ def conflicts(self, host, name, id, new_props, ignore_nulls=False):
                     new == str(omero.constants.GLACIER2PORT)):
                 continue
             elif old != new:
-                conflicts += (key + (":%s!=%s;" % (old, new)))
+                if conflicts != "":
+                    conflicts += "; "
+                conflicts += (key + (": %s!=%s" % (old, new)))
```

Comment:
Very nit-picky, but since you're modifying this line `("%s: %s!=%s" % (key, old, new))` would be clearer